client/components/checklist/checklist.js

```diff
@@ -129,8 +135,23 @@ export default class Checklist extends PureComponent {
 							onTaskClick: () => this.setExpandedTask( realIndex ),
 						} );
 					} ) }
+
+					{ wpAdminUrl && showAdminFooter && (
+						<JetpackChecklistFooter
+							wpAdminUrl={ wpAdminUrl }
+							handleWpAdminLink={ this.handleWpAdminLink }
+						/>
+					) }
+
+					{ completed > 0 && (
```

Comment:
Should this title be hidden when everything is completed, as it's kinda obvious visually?

<img width="837" alt="Screenshot 2019-07-08 at 09 22 17" src="https://user-images.githubusercontent.com/87168/60787702-60d64480-a163-11e9-8f28-1a8f67086899.png">

Reply:
Good call. I can drop the title when the checklist is completed.